modules/xfem/doc/content/source/reporters/ParisLaw.md

```diff
@@ -0,0 +1,15 @@
+# ParisLaw
+
+## Description
+
+The ParisLaw Postprocessor computes the crack extension size at all active crack front points in a fatigue crack growth scenario. Crack front points are provided by the [CrackMeshCut3DUserObject.md] UserObject and fracture integral values are provided by the [InteractionIntegral.md] vector postprocessor set-up by the [DomainIntegralAction.md].  The code is based on the Paris law and requires Paris law parameters [!param](/Reporters/ParisLaw/paris_law_c) and [!param](/Reporters/ParisLaw/paris_law_m) as the input. The amounts of crack extension at the crack front nodes are scaled such that the point with the largest effective K will have an extension equal to [!param](/Reporters/ParisLaw/max_growth_size).
```

Comment:
```suggestion
The ParisLaw Reporter computes the crack extension size at all active crack front points in a fatigue crack growth scenario. Crack front points are provided by the [CrackMeshCut3DUserObject.md] UserObject and fracture integral values are provided by the [InteractionIntegral.md] vector postprocessor, which can be set up by the [DomainIntegralAction.md].  The code is based on the Paris law and requires Paris law parameters [!param](/Reporters/ParisLaw/paris_law_c) and [!param](/Reporters/ParisLaw/paris_law_m) as the input. The amounts of crack extension at the crack front nodes are scaled such that the point with the largest effective K will have an extension equal to [!param](/Reporters/ParisLaw/max_growth_size).
```

Reply:
Is this reporter and the StressCorrosionCracking reporter only for 3D (hence the reference to CrackMeshCut3DUserObject)?